LCM/dsc/engine/ConfigurationManager/MSFT_DSCLocalConfigurationManager.c

```diff
@@ -22,8 +22,9 @@
 #include "EngineHelper.h"
 #include "EventWrapper.h"
 #include "Resources_LCM.h"
-
-
+#include <signal.h>
+// Variable to store old Signal for SIGCHLD
+struct sigaction oldAction ;
```

Comment:
why not wrap all references to oldAction in #if defined(BUILD_OMS). seems like for DYI it is initialized and never used.

Reply:
good idea. In fact that was the intention.